.github/workflows/ci.yml

```
@@ -27,7 +27,7 @@ jobs:
           mkdir ~/.shellcheck
           unzip shellcheck-stable.zip -d ~/.shellcheck
           mv ~/.shellcheck/shellcheck-stable.exe ~/.shellcheck/shellcheck.exe
-          echo "::add-path::$HOME/.shellcheck"
+          echo "${HOME}/.shellcheck" >> ${GITHUB_PATH}
```

This code is running if `matrix.os == 'windows-latest'`. So this is PowerShell rather than Bash.

```suggestion
          echo "$HOME/.shellcheck" >> $env:GITHUB_PATH
```